lsl_autobids/generate_dataset_json.py

```diff
@@ -1,21 +1,13 @@
 import json
-import toml
+import logging
 import os
-import yaml
-from globals import project_root
+from config_globals import project_root
+from utils import read_toml_file
 
-def read_toml_file(toml_file):
-    with open(toml_file, 'r') as file:
-        return toml.load(file)
+# Set up logging
```

Comment:
So we need that in all files, or is it possible to move that to __inot__? I don't know python enough